git/git.go

```diff
@@ -18,20 +20,23 @@ func NewGit(shell shell.Shell) Git {
 	return &RealGit{shell}
 }
 
-func (g *RealGit) ShowTopLevel(path string) (bool, string, error) {
-	out, err := g.shell.Cmd("git", "-C", path, "rev-parse", "--show-toplevel")
+func (g *RealGit) GitRoot(path string) (bool, string, error) {
+	out, err := g.shell.Cmd("git", "-C", path, "worktree", "list")
```

Comment:
Let's keep the git package clean and simply add a WorkTreeList command here, then determine of the name in [namer/git](https://github.com/joshmedeski/sesh/blob/476dbf94026d367d145c9f1ccc88bc8078cc8660/namer/git.go?plain=1#L51)